src/applications/claims-status/components/AddFilesFormOld.jsx

```diff
@@ -0,0 +1,344 @@
+import PropTypes from 'prop-types';
```

Comment:
Updated imports for @department-of-veterans-affairs and renamed the file, but otherwise didnt change anything here!